.github/workflows/build_prs_test_report.yaml

```diff
@@ -0,0 +1,46 @@
+name: TestReport
+on:
+  pull_request:
+    paths-ignore:
+      - website/**
+      - coverage/**
+
+jobs:
+  testreport:
+    runs-on: ubuntu-latest
+    services:
+      # Label used to access the service container
+      postgres:
+        # Docker Hub image
+        image: postgres
+        # Provide the password for postgres
+        env:
+          POSTGRES_PASSWORD: postgres
+        # Set health checks to wait until postgres has started
+        ports:
+          - 5432:5432
+        options: >-
+          --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
+          cache: 'yarn'
+      - run: yarn install --frozen-lockfile --ignore-scripts
+      - run: yarn build
+      - run: yarn build:frontend
```

Comment:
This is out of date, build is already building the frontend, but if we want to build frontend, I think we also need to remove --ignore-scripts (I'm not sure why we have them)